Enterprise-CMCS/cmcs-eregulations
# EREGCSC-1921 -- Statute SPA/Table

Resolves [EREGCSC-1921](https://jiraent.cms.gov/browse/EREGCSC-1921)

**Description**

Creates Statute Table that loads data from all SSA titles into a single table.

**This pull request changes:**

- Makes `/statutes` a Single Page App to support future enhancements
- Adds Header/Footer and Banner
- Pulls Statute data from API endpoint at `v3/statutes`
- Renders API response in a table that matches [Figma](https://www.figma.com/file/mXCFChrIR50gbFAvXV0kgB/statute-to-USC-conversion?node-id=60%3A618&mode=dev)
- Styles table to match mobile and desktop Figma designs, as well as implements behavior notes from Figma

**Steps to manually verify this change:**

1. Visit [Experimental Deployment and go to the `/statutes` page](https://371wnfkrul.execute-api.us-east-1.amazonaws.com/dev856/statutes/)
2. View page in desktop widths (greater than 1024px) and narrow widths (less than 1024px) and make sure table matches Figma 
3. Poke around table and make sure links work, things look good, etc.
